neqo-server/src/main.rs

```diff
@@ -91,66 +91,66 @@ impl Display for ServerError {
     }
 }
 
-#[derive(Debug, StructOpt)]
-#[structopt(name = "neqo-server", about = "A basic HTTP3 server.")]
+#[derive(Debug, Parser)]
+#[command(author, version, about, long_about = None)]
 struct Args {
     /// List of IP:port to listen on
-    #[structopt(default_value = "[::]:4433")]
+    #[arg(default_value = "[::]:4433")]
     hosts: Vec<String>,
 
-    #[structopt(name = "encoder-table-size", long, default_value = "16384")]
+    #[arg(name = "encoder-table-size", long, default_value = "16384")]
     max_table_size_encoder: u64,
 
-    #[structopt(name = "decoder-table-size", long, default_value = "16384")]
+    #[arg(name = "decoder-table-size", long, default_value = "16384")]
     max_table_size_decoder: u64,
 
-    #[structopt(short = "b", long, default_value = "10")]
+    #[arg(short = 'b', long, default_value = "10")]
     max_blocked_streams: u16,
 
-    #[structopt(
-        short = "d",
+    #[arg(
+        short = 'd',
         long,
         default_value = "./test-fixture/db",
-        parse(from_os_str)
+        value_parser=clap::value_parser!(PathBuf)
```

Comment:
```suggestion
```

Why are these explicit value parser definitions needed? Just played around with it and it works without for me locally.